controllers/interviews.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs')
+const Interviews = require('../data/interviews.json')
+
+const listInterviews = (req, res) => {
+    res.status(201).json(Interviews)
```

Comment:
Status 201 is for creations. 
https://developer.mozilla.org/es/docs/Web/HTTP/Status/201